### ISSUE TITLE
BUG : VS2017 Designer Editor | Selection Issue

### ISSUE THREAD

**Issue by LinkinForEver** [ButchersBoy/Dragablz#191]
Hi , in vs2017 when you create a Dragablz in your window , you can't select anything else !
I updated visual studio to latest version but nothing is different ...
Video Attached : <a href="http://www18.zippyshare.com/v/lVELSScw/file.html" target="_blank" class="zippyshare_link">bug.mp4</a>

**Comment by ButchersBoy** [comment]
Yes, this is a known issue.  The easiest thing is for you to break your tab content into individual UserControls so you can work with them in the designer in isolation.

**Comment by LinkinForEver** [comment]
@ButchersBoy thanks for response , I use VS2012 for tab designs and switch on VS2017 , It's okay but It'll be cool if you fix it in future .
Regards
